### ISSUE TITLE
Show Default Value that is not in the  dropdown list.

### ISSUE THREAD

**Issue by DurgeshSupraDigit** [hossein-zare/react-native-dropdown-picker#125]
Hello ,
I need to display a default value which is not present in the list ,
it gives "choice.label" error , from which i understand that  the default value  must be present in dropdown list
which in my case is not present in the dropdown list.
.

However , is it possible to display a  default value which is not present in the dropdown list ?

**Comment by Rahul1d** [comment]
You can use placeholder.

**Comment by hossein-zare** [comment]
Hello @DurgeshSupraDigit
**This feature will be available in the next version.**

```javascript
this.state = {
    country: 'uk'
}

items={[
    {label: 'UK', value: 'uk', hidden: true},
    ...
]}
defaultValue={this.state.country}
```

**Comment by DurgeshSupraDigit** [comment]
> You can use placeholder.

Yes for the time being , i m using this as a work around !!

**Comment by gudipudiprashant** [comment]
Hi @hossein-zare Is this still available in 5.4? I don't see this property. If not, is there a workaround? 

My use-case is to show a list of items, and when users type in the search box and don't see an item they want, they should see a default item - "Other". Is this possible?
